src/components/dropdown/dropdown.tsx

```diff
@@ -163,6 +166,11 @@ export class Dropdown implements InteractiveComponent, OpenCloseComponent, Float
     this.mutationObserver?.observe(this.el, { childList: true, subtree: true });
     this.setFilteredPlacements();
     this.reposition();
+    const isOpen = this.active || this.open;
```

Comment:
Can you check if the property is set from the attribute here? If not, I think this also needs to look at the user-set attributes.

Reply:
yup, the property is set from the attribute.